app/views/content/help-and-advice.md

```diff
@@ -41,24 +41,22 @@ An explore teaching adviser can support you if you're an undergraduate student a
 
 Our events are a great way to meet teachers and other experts to talk about any concerns or questions you have about becoming a teacher.
 
-### Online question and answer sessions
-
-There are online, text-based sessions where you can ask a panel of specialists the questions that matter to you. These sessions, run by DfE, cover a range of different topics so you can post specific questions and receive advice tailored to your particular circumstances.
-
-### Training provider events
-
-You could also attend a training provider event, either online or in person, and hear directly from teacher training providers about the courses they offer and how to apply.
-
-### Train to Teach events
+### Get Into Teaching events
 
-Train to Teach events are run by the Department for Education (DfE) and are free to attend. You'll be able to:
+Get Into Teaching events are run by the Department for Education (DfE) and are free to attend. You'll be able to:
```

Comment:
```suggestion
<a href="/events/about-get-into-teaching-events">Get Into Teaching events</a> are run by the Department for Education (DfE) and are free to attend. You'll be able to:
```

Although we don't have other inline links on this page, I think it is worth adding one for GIT events for now (this page will soon be superseded by a category page anyway)